### PR TITLE
krimdok2: journal & news updates

### PR DESCRIPTION
**Pull request by mtrojan-ub**

### CODE REVIEW

File: themes/krimdok2/templates/content/journals_de.phtml [hunk 1]
```diff
@@ -4,6 +4,7 @@
     Um den Zugang zu laufenden, kriminologisch relevanten Open-Access-Volltexten zu vereinfachen, wurden thematisch passende lizenzfreie Zeitschriften in einer Übersicht zusammengefasst. Weitere Informationen zu gegenwärtig erscheinenden Zeitschriften sowie zu der Auswertung von Einzelbeiträgen für KrimDok finden Sie in den unten genannten Listen.
 </p>
 <ul>
-    <li><a href="/krimdok_docs/Journals_OA.pdf">Liste kriminologisch relevante Open-Access-Zeitschriften (pdf)</a></li>
     <li><a href="/krimdok_docs/Journals.pdf">Liste der derzeit für KrimDok ausgewerteten Zeitschriften (pdf)</a></li>
+    <li><a href="/krimdok_docs/Journals_OA.pdf">Liste kriminologisch relevante Open-Access-Zeitschriften (pdf)</a></li>
```
Comment (ruschein) [inline themes/krimdok2/templates/content/journals_de.phtml:8]: "r" is missing.

File: themes/krimdok2/templates/content/journals_en.phtml [hunk 2]
```diff
@@ -4,6 +4,7 @@
     To facilitate access to open-access documents relevant to criminological research, field specific and non-licensed journals are summarized in a list. Further information to current journals is available in the following lists:
 </p>
 <ul>
-    <li><a href="/krimdok_docs/Journals_OA.pdf">List of currently processed journals in KrimDok (pdf)</a></li>
     <li><a href="/krimdok_docs/Journals.pdf">List of criminological open-access journals (pdf)</a></li>
+    <li><a href="/krimdok_docs/Journals_OA.pdf">List of currently processed journals in KrimDok (pdf)</a></li>
+    <li><a href="/krimdok_docs/Journals_Nationallizenz.pdf">List of criminological journals with national licence (pdf)</a></li>
```
Comment (ruschein) [inline themes/krimdok2/templates/content/journals_en.phtml:9]: I prefer "with *a* national"...

File: themes/krimdok2/templates/content/journals_de.phtml (hunk 1 above)
Comment (jriedl) [inline themes/krimdok2/templates/content/journals_de.phtml:8]: Missing 'r' at "relevante"?

File: themes/krimdok2/templates/content/news_fid.phtml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <h1>News</h1>
 <ul>
+    <li><b><u>09.04.2020</u></b>: Die <a href="<?=$this->url('content-page', ['page' => 'journals'])?>">Liste der aktuell für KrimDok ausgewerteten Zeitschriften</a> wurde aktualisiert. Zudem gibt es nun auch eine Übersicht, welche kriminologisch relevanten Zeitschriften eine Nationallizenz haben. Beide Listen sowie die Übersicht über kriminologisch relvante Open-Access-Zeitschriften finden sich im Panel <a href="<?=$this->url('content-page', ['page' => 'krimdok'])?>">Einzelheiten zu KrimDok</a> unter <a href="<?=$this->url('content-page', ['page' => 'krimdok'])?>#contributions">Zeitschriften und Schriftenreihen</a>.</li>
```
Comment (ruschein) [inline themes/krimdok2/templates/content/news_fid.phtml:3]: relvante -> relevante

Comment (jriedl) [inline themes/krimdok2/templates/content/news_fid.phtml:3]: "relvante"

**Pull request merged**